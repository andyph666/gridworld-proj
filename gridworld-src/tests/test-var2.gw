main{
	string i = "boss";
	string j = "enemy";

	i = "boss1";
	j = "enemy1";

	print(i);
	print(j);
}