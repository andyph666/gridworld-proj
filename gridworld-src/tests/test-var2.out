boss1
enemy1